src/main/java/com/powsybl/openloadflow/network/LfShunt.java

```diff
@@ -19,5 +19,7 @@
 
```

Comment:
If we have access to the bus voltage in the `LfShunt`, we don't need `q` anymore. @geofjamg what do you think ?